### ISSUE TITLE
Double click to select entire procedure

### ISSUE THREAD

**Issue by KarlSevenSeven** [twinbasic/twinbasic#1353]
**Is your feature request related to a problem? Please describe.**
In VB6 we can double click before a procedure declaration to select the whole procedure.

**Describe the solution you'd like**
Same as in VB6.

**Describe alternatives you've considered**
Click before eg. Sub, scroll to End Sub, Shift click after Sub.

**Additional context**
I didn't know this for nearly a quarter century, and learned this a few weeks ago in vbforums.

**Comment by sokinkeso** [comment]
I didn't know this for nearly a quarter century too, and learned it a few minutes ago from you!😂
It is a nice and useful feature, and I would like to extend it more.. since tB has collapsable areas in the code (functions, properties, regions ..) , the selection could be on any of these areas. (select a whole region or select a whole function etc)

**Comment by WaynePhillipsEA** [comment]
I didn't know this until just now.  Mind blown.

**Comment by dmrvb** [comment]
I can't make this work, exactly where do I do the double click?

**Comment by sokinkeso** [comment]
Between the area of the breakpoints and the code pane

**Comment by dmrvb** [comment]
Thankyou, this is new to me too. 
In general I am the sort of person who likes to get up and running and using software as quickly as I can so I don't take the time to learn all the shortcuts and hidden features. There are many like me so I try hard not to build hidden features into the software that I write. When I do include hidden things then they are for my own use/testing, and after a few months I can't remember how to get to them. Shortcut keys are good, but their functionalirty should also be availabe from a menu or similar, or at least well documented. A lesson for tB here :-)

**Comment by sokinkeso** [comment]
> cumented. A lesson for tB here :-)

I agree..I do the same for my software too...but this VB6 feature was not hidden...we just did not take the time to read the entire manual...😉

**Comment by dmrvb** [comment]
To select a complete sub I switch to the "display single sub/function" mode in VB6 then do a select all.
I find the ability to display a single sub in the edit window is really useful, almost essential, and hope Wayne can find a way to do thisin tB one day.   (how do I insert a proper smiley in GitHub??)

**Comment by sokinkeso** [comment]
(how do I insert a proper smiley in GitHub??), with right click in the textbox

**Comment by rexxitall** [comment]
Autocad has way more then over 1000 functions. I use the menubar if i have forgotten a function. Its a handy reference. Keybord shortcuts comes only in handy if there is a shortcut editor. Autocad has such a thing as well. But this i think should all delayed after the compiler itselv ist ready feature complete and stable. :)
